library/Zend/View/Helper/HtmlTag.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2014 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Do I really have to modify that? :) I also have some other PRs planned for 2.4.0 within which new classes are added, like this one: https://github.com/zendframework/zf2/pull/6058, and I haven't got such remark from a code reviewer.